guides/v2.3/architecture/security_intro.md

```diff
@@ -8,7 +8,7 @@ menu_order:
 
 ## Enhanced password management
 
-Magento has strengthened the hashing algorithms (SHA-256) used in password management. In addition, we added support for Argon2ID13, which is used by default in PHP 7.2 or if you have installed the Libsodium extension for PHP.
+Magento has strengthened the hashing algorithms (SHA-256) used in password management. Magento now supports Argon2ID13 through the PHP sodium extension. Sodium version 1.0.13 or higher is required for this functionality.
```

Comment:
Hi @hguthrie & @dobooth 

Be aware that there is a difference between the php sodium extension and the libsodium library.
The php sodium extension has a dependency on libsodium, so they both need to be installed.

In this case, the libsodium library needs to be at least version 1.0.13, the php sodium extension version doesn't really matter and probably just follows the php version numbering.

Hope this helps :)

Reply:
Yes, @hostep ! Thank you. 

```suggestion
Magento has strengthened the hashing algorithms (SHA-256) used in password management. Magento now supports Argon2ID13 through the PHP sodium extension, which requires the libsodium library version 1.0.13 or higher.
```

Reply:
Looks good, thanks!